src/app/authentication/security-http-interceptor.ts

```diff
@@ -10,7 +10,8 @@ export class SecurityHttpInterceptor implements HttpInterceptor {
 
   intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
     const accessToken = this.authenticationService.getToken();
-    const authRequest = request.clone({setHeaders: {Authorization: `Bearer ${accessToken}`}});
+    // TODO: I think it would be better is authenticationService had an isLoggedIn method to determine if we add the auth token
```

Comment:
We can easily add a helper getter `isLoggedIn()`. I would say we should remove this comment though and add an issue instead, keeps the code from getting sprinkled with comments.

Reply:
Good call I will remove it and add an issue